sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -154,15 +227,33 @@ export interface IssuerParameters {
     name?: string;
 }
 
+// @public
+export type KeyCurveName = "P-256" | "P-384" | "P-521" | "P-256K";
```

Comment:
I wonder if these should come from keyvault-keys. Certificates already depends on it.

Reply:
Woah! that's interesting. We could. Not sure if we should!

Reply:
If its the same value and this type is used in conjunction with features from the keys library, then I would say that we should import it from the keys library and then re-export it from certificates

Reply:
We haven't taken an explicit dependency on keyvault-keys. Are we sure we want to do this? Right now it's a devDependency.

Reply:
Let's keep it like it is in this PR for now. We can follow up next week if needed.